bundle/Resources/config/services/layout_resolver.yaml

```diff
@@ -44,6 +44,25 @@ services:
         tags:
             - { name: netgen_layouts.target_type.doctrine_handler, target_type: sylius_taxon }
 
+    netgen_layouts.sylius.layout_resolver.target_type.page:
+        class: Netgen\Layouts\Sylius\Layout\Resolver\TargetType\Page
+        arguments:
+            - "%netgen_layouts.sylius.target.pages%"
+        tags:
+            - { name: netgen_layouts.target_type, priority: 170 }
+
+    netgen_layouts.sylius.layout_resolver.target_handler.doctrine.sylius_page:
+        class: Netgen\Layouts\Sylius\Layout\Resolver\TargetHandler\Doctrine\Page
+        tags:
+            - { name: netgen_layouts.target_type.doctrine_handler, target_type: sylius_page }
+
+    netgen_layouts.sylius.layout_resolver.target_type.form_mapper.page:
```

Comment:
Please move it up before the target handler to satisfy my OCD, as all other sets of target type services are ordered that way!

Reply:
@emodric I don't understand this one? Everything looks correct to me.